erigon-lib/txpool/pool.go

```diff
@@ -1551,10 +1532,8 @@ func promote(pending *PendingPool, baseFee, queued *SubPool, pendingBaseFee uint
 			tx := pending.PopWorst()
 			announcements.Append(tx.Tx.Type, tx.Tx.Size, tx.Tx.IDHash[:])
 			baseFee.Add(tx, logger)
-		} else if worst.subPool >= QueuedPoolBits {
```

Comment:
Don’t understand why this line removed

Reply:
are you sure that QueuedPoolBits has only 1 bit?  

Even if it’s true - still “bad txs need move to queued sub-pool” because maybe they will improve in future (sender may increase it’s balance, etc…)

Reply:
ok, make sense for me now